app/views/templates/rulingCoverLetterTemplate.scala.html

```diff
@@ -217,7 +217,7 @@ <h4 class="heading-small">@messages("pdf.ruling.section.communicatingByEmail.tit
             <p class="govuk-body-s">
                 @messages("pdf.ruling.section.communicatingByEmail.copy")
             </p>
-            <h4 class="heading-small">@messages("pdf.ruling.section.acknowledgement.title")</h4>
+            <h4 class="govuk-heading-s">@messages("pdf.ruling.section.acknowledgement.title")</h4>
             <p class="govuk-body-s">
                 @Html(messages("pdf.ruling.section.acknowledgement"))
```

Comment:
I had a convo with Jess and she confirmed that [these lines](https://github.com/hmrc/binding-tariff-trader-frontend/blob/DDCE-5412/app/views/templates/rulingCoverLetterTemplate.scala.html#L224-L232) should also be removed. Please remove `pdf.ruling.section.yours` and `pdf.ruling.section.hmrctcs` in both messages files [here](https://github.com/hmrc/binding-tariff-trader-frontend/blob/DDCE-5412/conf/messages.en#L536-L537) and [here](https://github.com/hmrc/binding-tariff-trader-frontend/blob/DDCE-5412/conf/messages.cy#L521-L522), thanks.